lib/flipper/adapters/active_record.rb

```diff
@@ -45,10 +45,11 @@ def features
 
       # Public: Adds a feature to the set of known features.
       def add(feature)
-        attributes = {key: feature.key}
         # race condition, but add is only used by enable/disable which happen
         # super rarely, so it shouldn't matter in practice
-        @feature_class.where(attributes).first || @feature_class.create!(attributes)
+        unless @feature_class.find_by_key(feature.key)
```

Comment:
I think dynamic finders are [deprecated](https://github.com/rails/activerecord-deprecated_finders) `find_by_key` which is why I went with the less aesthetically pleasing `where(...).first`. Would you mind swapping this back to the where so it is more future proof?